src/pages/Peers.tsx

```diff
@@ -88,7 +88,7 @@ export const PeersPage = (): JSX.Element => {
                           connection.getMetadata({ nodeNum: node.data.num }),
                           {
                             loading: "Requesting Metadata...",
-                            success: "Recieved Metadata",
```

Comment:
Could you please revert this change, I have some rework to this page uncommitted,  I'll fix it there